index.html

```diff
@@ -9,76 +9,259 @@
+
+
+    <recentwork id="recentwork"><p id="recentwork_text">My Recent Works</p>
+    <img src="./assets/Vector.svg" alt="divider">
+
+    <div id="frame44">
+        <div id="featured_img">
+            <div>
+            <img src="./assets/Img Placeholder.svg" alt="work featured image"></div>
+            </div>
+
+        <div id="right_block">
+            <p id="RB_title_post">Multi-Post Stories</p>
+            <p id="RB_secondary_text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul id="RB_tags">
+                <li class="RB_tags"><p class="RB_tags_text">css</p></li>
+                <li class="RB_tags"><p class="RB_tags_text">html</p></li>
+                <li class="RB_tags"><p class="RB_tags_text">bootstrap</p></li>
+                <li class="RB_tags"><p class="RB_tags_text">ruby</p></li>
+            </ul>
+            <a href="https://github.com/glitcher255/micro-porfolio"><btn_enable id="btn_enable" class="orange_btns"><p id="btn_enable_text">See Project</p></btn_enable></a>
+          </div>
+          
+
+        <ul id="frame43">
+          <div id="card_works">
+            <div id="action">
+              <img src="./assets/Mask Group.png" alt="Professional art data">
+            </div>
+            <a href="https:github.com"><div id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </div></a>
+
+            <li id="card_data">
+              <div id="card_title"><p>Profesional Art Printing Data</p></div>
+              <div id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+            </li>
+          </div>
+
+
+          <div id="card_works">
+            <div id="action">
+              <img src="./assets/Mask Group.png" alt="Professional art data">
+            </div>
+            <a href="https:github.com"><div id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </div></a>
+
+            <li id="card_data">
+              <div id="card_title"><p>Profesional Art Printing Data</p></div>
+              <div id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
```

Comment:
Kindly, in previous reviews I told you only use valid HTML labels, and turn them into correct or divs, I attached documentation about HTML and its correct uses. (Check all your code to remove them all) https://www.tutorialspoint.com/html5/html5_tags.htm